tests/cpptests/test_cpp_index.cpp

```diff
@@ -438,14 +439,16 @@ TEST_F(IndexTest, DISABLED_testOptional) {
 }
 
 TEST_F(IndexTest, testNumericInverted) {
-
-  InvertedIndex *idx = NewInvertedIndex(Index_StoreNumeric, 1);
+  size_t size = 0;
+  InvertedIndex *idx = NewInvertedIndex(Index_StoreNumeric, 1, &size);
 
   for (int i = 0; i < 75; i++) {
     size_t sz = InvertedIndex_WriteNumericEntry(idx, i + 1, (double)(i + 1));
-    // printf("written %zd bytes\n", sz);
-
-    ASSERT_TRUE(sz > (i ? 1 : 0)); // first doc has zero delta (not written)
+    if(i < 3) {
+      ASSERT_TRUE(sz == 0); // first doc has zero delta (not written)
+    } else if(i > 2 && i < 6 ) {
```

Comment:
Can check beyond the first 6 values (and can explain that each value has an additional byte for a header and an additional byte for a doc id delta (except for the first value in a block))

Reply:
OK. Now the tests is simulating the buffer growth to get the expected size.